assets/script.js

```diff
@@ -55,6 +62,11 @@ function getJoke() {
 	return fetchData(jokeUrl);
 }
 
+function getQuote() {
+	const quoteUrl = '';
```

Comment:
you need to have the quoteUrl here, otherwise the script is trying to fetch from an empty string